code-review.js

```diff
@@ -1,60 +1,119 @@
+"use strict";
 /**
  * A priority queue stores a list of items but each can have a numeric priority value.
  * Items with a higher priority are dequeued before items with a lower priority.
  * Implemented as a hash of arrays where the hash keys are priority values.
+ *
+ * @class
  */
-function PriorityQueue(size) {
-	this.store = {};	// keys are priorities, values are arrays of elements
+function PriorityQueue() {
+	this.queue = {};	// keys are priorities, values are arrays of elements

```

Comment:
Thought it made more sense naming this `queue` instead of `store` since this is a PriorityQueue after all!